### DISCUSSION TITLE
Anyone having problem posting image on home and weird behavior?

### DISCUSSION THREAD

**Discussion by zxcvvvvbanan** [vuejs/vitepress#4344] [Q&A] [answered]
My index.md is not special at all. It seems just fine and when I build in local dev, it shows an image. However, when I push it to github pages and deployment is done, image never shows up.

I tried the following and none of these two worked. "./image.png" "/image.png".
I even replace the image to vitepress default .webp image and removed any script associated with it.

_One more thing, my home behavior got weird. It repeats the contents twice._   <-Solved. 

`---
layout: home
hero:
  name: <name>
  image:
     src: <path>
`

**Comment by brc-dd** [comment, marked as answer]
Store your image in public directory. Assets referenced from frontmatter aren't automatically processed.

**Reply by zxcvvvvbanan** [reply]
This worked :) Thanks !
